### DISCUSSION TITLE
Modal inside modal

### DISCUSSION THREAD

**Discussion by ritvij14** [saadeghi/daisyui#2093] [Q&A]
I have a particular use case where I need to trigger the opening of a modal from another modal.

Code example:-

```typescript
const ModalA: React.FC = () => {
  const [openModalB, setOpenModalB] = useState(false);
  return (
    <div className="modal modal-open">
        <div className="modal-box rounded-xl border border-[#00000026] bg-white p-8">
          <button
              onClick={() => {
                setOpenModalB(true);
              }}
            >
              Create new&nbsp; {NewCollection("#D1530B")}
            </button>
      </div>
      {openModalB && (
        <ModalB
          close={() => {
            setOpenModalB(false);
          }}
        />
      )}
    </div>
  );
};
```

And `ModalB` is a smaller sized modal similar to `ModalA`

But when I try to close `ModalB` using a button click to execute the `close` function, I get this warning in my console and the second modal doesn't close.
<img width="509" alt="image" src="https://github.com/saadeghi/daisyui/assets/51456744/d97fa7d4-32c2-4fff-9acd-13a2cec9ff3a">

@saadeghi any suggestions as to what to do in this scenario?

**Comment by saadeghi** [comment]
This is not related to daisyUI. daisyUI only provides the visual style for the modal.  

However, if you provide an example repo on Stackblitz or CodeSandbox or a minimal GitHub repo, I can help you with that.  
Currently I don't know what is in `ModalB` component or how is your `close` function so it's really hard to guess the issue.
